hubspot/plugin.py

```diff
@@ -34,6 +35,348 @@ class HubSpotContact:
+        "mobilephone",
+        "phone",
+        "fax",
+        "hs_analytics_first_timestamp",
+        "hs_email_last_email_name",
+        "hs_email_last_send_date",
+        "hs_social_facebook_clicks",
+        "address",
+        "engagements_last_meeting_booked",
+        "engagements_last_meeting_booked_campaign",
+        "engagements_last_meeting_booked_medium",
+        "engagements_last_meeting_booked_source",
+        "hs_analytics_first_visit_timestamp",
+        "hs_email_last_open_date",
+        "hs_latest_meeting_activity",
+        "hs_sales_email_last_replied",
+        "hs_social_linkedin_clicks",
+        "hubspot_owner_id",
+        "notes_last_contacted",
+        "notes_last_updated",
+        "notes_next_activity_date",
+        "num_contacted_notes",
+        "num_notes",
+        "owneremail",
+        "ownername",
+        "surveymonkeyeventlastupdated",
+        "webinareventlastupdated",
+        "city",
+        "hs_analytics_last_timestamp",
+        "hs_email_last_click_date",
+        "hs_social_google_plus_clicks",
+        "hubspot_team_id",
+        "linkedinbio",
+        "twitterbio",
+        "hs_all_owner_ids",
+        "hs_analytics_last_visit_timestamp",
+        "hs_email_first_send_date",
+        "hs_social_num_broadcast_clicks",
+        "state",
+        "hs_all_team_ids",
+        "hs_analytics_source",
+        "hs_email_first_open_date",
+        "hs_latest_source",
+        "zip",
+        "country",
+        "hs_all_accessible_team_ids",
+        "hs_analytics_source_data_1",
+        "hs_email_first_click_date",
+        "hs_latest_source_data_1",
+        "linkedinconnections",
+        "hs_analytics_source_data_2",
+        "hs_email_is_ineligible",
+        "hs_language",
+        "hs_latest_source_data_2",
+        "kloutscoregeneral",
+        "hs_analytics_first_referrer",
+        "hs_email_first_reply_date",
+        "jobtitle",
+        "photo",
+        "hs_analytics_last_referrer",
+        "hs_email_last_reply_date",
+        "message",
+        "closedate",
+        "hs_analytics_average_page_views",
+        "hs_email_replied",
+        "hs_analytics_revenue",
+        "hs_lifecyclestage_lead_date",
+        "hs_lifecyclestage_marketingqualifiedlead_date",
+        "hs_lifecyclestage_opportunity_date",
+        "lifecyclestage",
+        "hs_lifecyclestage_salesqualifiedlead_date",
+        "createdate",
+        "hs_lifecyclestage_evangelist_date",
+        "hs_lifecyclestage_customer_date",
+        "hubspotscore",
+        "company",
+        "hs_lifecyclestage_subscriber_date",
+        "hs_lifecyclestage_other_date",
+        "website",
+        "numemployees",
+        "annualrevenue",
+        "industry",
+        "associatedcompanyid",
+        "associatedcompanylastupdated",
+        "hs_predictivecontactscorebucket",
+        "hs_predictivecontactscore"
+    ]
+
+
+@dataclass
+class HubSpotContactCustomPropertyName:
+    name: str
```

Comment:
Does this structure buy much for custom properties?  It doesn't provide much type checking benefit and only adds to the code that code gen needs to write.